tests/test_enums.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from algobot.enums import OrderType
+
+
+class OrderTypeTest(unittest.TestCase):
```

Comment:
let's use pytest? We should soon start rewriting all unit test tests to leverage pytest :p